src/test/routes.test.ts

```diff
@@ -0,0 +1,360 @@
+import mongoose from "mongoose";
+import request from "supertest";
+
+import { app, server } from "../index";
+import * as _routesData from "./routes.test.json";
+
+interface Route {
+  name: string;
+  postData?: Record<string, unknown>;
+  putData?: Record<string, unknown>;
+  hasDelete: boolean;
+  hasGetAll: boolean;
+  hasGetById: boolean;
+  hasCreate: boolean;
+  hasUpdate: boolean;
+}
+
+beforeAll(() => {
+  jest.spyOn(console, "warn").mockImplementation(jest.fn());
+});
+
+const routes: (Route & {
+  createdId?: string;
+})[] = _routesData.data;
+
+let refreshToken: string;
+let accessToken: string;
+let id: string;
+const register = (data: object) => {
+  return request(app).post("/api/auth/register").send(data);
+};
+describe("Routes tests", () => {
+  describe("User routes", () => {
+    describe("Auth routes", () => {
+      describe("Register", () => {
+        it("should work", async () => {
+          await register({
```

Comment:
we can get problem if previous test was bad at position "delete" and such user was at database